glitter/blocks/image/admin.py

```diff
@@ -26,6 +32,25 @@ def formfield_for_dbfield(self, db_field, **kwargs):
             )
         return formfield
 
+    def get_urls(self):
+        urls = super(ImageBlockAdmin, self).get_urls()
+        image_block_urls = [
+            url(r'^get-lazy-images/$', self.get_lazy_images, name='get-lazy-images')
+        ]
+        return image_block_urls + urls
+
+    def get_lazy_images(self, request):
+        last_image_id = request.GET['last_image_id']
```

Comment:
Could we harden this up a bit?

If I go to http://127.0.0.1:8000/blockadmin/glitter_image/imageblock/get-lazy-images/ - I get:

```
MultiValueDictKeyError at /blockadmin/glitter_image/imageblock/get-lazy-images/
"'last_image_id'"
```

And if I go to: http://127.0.0.1:8000/blockadmin/glitter_image/imageblock/get-lazy-images/?last_image_id=bleh I get:

```
ValueError at /blockadmin/glitter_image/imageblock/get-lazy-images/
invalid literal for int() with base 10: 'bleh'
```

My personal preference would be to use Django pagination instead of all this, however just hardening the code here would be fine.